default/stringtables/en.txt

```diff
@@ -5957,77 +5957,47 @@ An Experimentor outpost exists in this system, and defeating them is necessary t
 PROBIOTIC_SPECIAL
 Probiotic Soup
 PROBIOTIC_SPECIAL_DESC
-'''The great diversity of unique single-celled organisms have applications to the health and maintenance of all types of organic life.
-
-[[encyclopedia ORGANIC_SPECIES_TITLE]] [[GROWTH_SPECIAL_POPULATION_INCREASE]] [[ORGANIC_SPECIES_TITLE]].'''
+The great diversity of unique single-celled organisms have applications to the health and maintenance of all types of organic life.
```

Comment:
Was it intentional to remove the extra explanatory macros here and below I put in late last year or was it just crossed over due to the long time this was open?